tests/pytests/test_erisieve.py

```diff
@@ -92,7 +93,7 @@ def test_schwarz_vs_csam():
         else:
             screen_count_none += 1
 
-    assert psi4.compare_values(75792, screen_count_both, 'Quartets CSAM Screened, Cutoff 0')  #TEST
-    assert psi4.compare_values(1344, screen_count_csam, 'Quartets CSAM Screened, Cutoff 0')  #TEST
-    assert psi4.compare_values(0, screen_count_schwarz, 'Quartets CSAM Screened, Cutoff 0')  #TEST
-    assert psi4.compare_values(27840, screen_count_none, 'Quartets CSAM Screened, Cutoff 0')  #TEST
+    assert psi4.compare_integers(75792, screen_count_both, 'Schwarz vs CSAM Screening, Cutoff 1.0e-12')  #TEST
+    assert psi4.compare_integers(1344, screen_count_csam, 'Schwarz vs CSAM Screening, Cutoff 1.0e-12')  #TEST
+    assert psi4.compare_integers(0, screen_count_schwarz, 'Schwarz vs CSAM Screening, Cutoff 1.0e-12')  #TEST
+    assert psi4.compare_integers(27840, screen_count_none, 'Schwarz vs CSAM Screening, Cutoff 1.0e-12')  #TEST
```

Comment:
you want to drop the `psi4.` here. `psi4.compare_integers` is the True/raise, while `compare_integers` (from tests/pytests/utils is the True/False. Also, no need for `#TEST`. (The `#TEST` flag is for lines to remove when generating `samples/` from `tests/` and that doesn't happen for the pytests.